(same for chimera)
--
From b353968eec5fcbd0f402562306f32ac381470254 Mon Sep 17 00:00:00 2001
From: =?UTF-8?q?S=C3=B6ren=20Tempel?= <soeren+git@soeren-tempel.net>
Date: Tue, 11 Jul 2023 15:41:42 +0200
Subject: [PATCH] go.env: Don't switch Go toolchain version as directed in
 go.mod

Alpine Linux only supports toolchain versions in $PATH and does not
support the Go 1.21 toolchain downloading feature. As it is unlikely
that users will have multiple toolchains installed in $PATH, switching
toolchains by default is undesirable.

Therefore, this patch restores the old pre <1.21 behavior where
the local toolchain was used by default even if a different version
was specified in go.mod.
---
 go.env | 8 +++++---
 1 file changed, 5 insertions(+), 3 deletions(-)

diff --git a/go.env b/go.env
index 6ff2b921d4..1ee5c16486 100644
--- a/go.env
+++ b/go.env
@@ -7,6 +7,8 @@
 GOPROXY=https://proxy.golang.org,direct
 GOSUMDB=sum.golang.org
 
-# Automatically download newer toolchains as directed by go.mod files.
-# See https://go.dev/doc/toolchain for details.
-GOTOOLCHAIN=auto
+# Don't automatically attempt to switch to a newer toolchain
+# as directed by go.mod files as Alpine does not support
+# toolchain downloading and hence can only retrieve toolchain
+# versions that are already in $PATH.
+GOTOOLCHAIN=local
